### ISSUE TITLE
what are the differences compared to normal better discord and list them

### ISSUE THREAD

**Issue by Pickymarker** [scythe-project/BetterDiscord-Plus#1]
what are the differences compared to normal better discord and list them

**Comment by scythe-project** [comment]
•Integrated store for plugins/themes
•Integrated emoji/sticker database
•Theme studio category
•Multi CSS editor category
•14 extra BetterDiscord settings
•Less lag (25% - 40%) when using themes
•Custom sounds (configurable)
•Message spammer
•Server nuke option
•Webhook multi tool category
•User management category
•Server management category
•Custom folder options
•Custom and assignable badges

Bottom line is, most options only available via plugins have been
Integrated into the client itself. The main goal of BD+ is to increase
stability and performance. BD+ will also be available on not just Windows
but Linux, MAC, Android AND IOS.

**Comment by Pickymarker** [comment]
Can you have the .asar file so that people don't have to use the installer if they already have betterdiscord so that I don't have to use the installer

**Comment by Pickymarker** [comment]
also, virus total lights up from giving it the exe.
